.eslintrc.cjs

```diff
@@ -56,31 +57,19 @@ module.exports = {
       {
         selector: "interface",
         format: ["PascalCase"],
-        custom: {
-          regex: "^I[A-Z]",
-          match: true,
-        },
```

Comment:
Distribution doesn't obey this rule, but I think it's probably ok to relax this one for now?

Reply:
I intend to suggest defaulting to `type` instead of `interface` whenever possible so that's a nudge in that direction.